CHANGES.md

```diff
@@ -4,7 +4,10 @@ Libplanet changelog
 Version 2.1.1
 -------------
 
-To be released.
```

Comment:
Eh, need to have a release date (or a placeholder).

Reply:
Oops, I didn't intend it 😱 I'll make a new pr for release.